modules/metering-cluster-capacity-examples.adoc

```diff
@@ -5,7 +5,7 @@
 [id="metering-cluster-capacity-examples_{context}"]
 = Measure cluster capacity hourly and daily
 
-The following Report demonstrates how to measure cluster capacity both hourly and daily. The daily Report works by aggregating the hourly Report's results.
+The following `Report` object demonstrates how to measure cluster capacity both hourly and daily. The daily report works by aggregating the hourly report's results.
```

Comment:
Does this need to be "`Report` custom resource`?

Reply:
I might actually change this to just "report" to be consistent with the other sections on this page.